portfolio/src/main/webapp/script.js

```diff
@@ -27,35 +27,26 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
+//Generates a number between 1 and 200
+function randomNumber() {
```

Comment:
There are two places where random numbers are required (one between [1...4], the other [1...200]). Can you make this function take an input parameter to specify the highest number so you can use it in both cases?

Reply:
Yeah that makes more sense.